app/scripts/modules/core/src/presentation/formsy/BasicFieldLayout.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { IFormFieldLayoutProps } from './formFieldLayout';
+
+/**
+ * A Form Field Layout component for Formsy Form Field components
+ *
+ * Accepts four react elements as props (_label, _input, _help, _error) and lays them out using bootstrap grid.
+ *
+ * +----------------div.form-group----------------------------+
+ * |                 +-------------div.col-md-9--------------+|
+ * |<label.col-md-3> |+-------------------------------------+||
+ * |                 ||input element                        |||
+ * |                 |+-------------------------------------+||
+ * |                 |                                       ||
+ * |                 |<help element>                         ||
+ * |                 |                                       ||
+ * |                 |<validation element>                   ||
+ * |                 |                                       ||
+ * |                 +---------------------------------------+|
+ * +----------------------------------------------------------+
+ */
+export class BasicFieldLayout extends React.Component<IFormFieldLayoutProps, {}> {
+  constructor(props: IFormFieldLayoutProps) {
+    super(props);
+  }
+
+  public render() {
+    const { _label, _input, _help, _error, showRequired, showError } = this.props;
```

Comment:
Oh good, you did notice the underscores!
I  used an underscore to indicate that these props are special, in that they should be react *elements* and not something like a plain string.

I found myself typing `<element label="Foo">` but that's wrong.  Frankly I don't like them either.  I'll take them out.  Typescript is good enough